tests/endpoints.test.ts

```diff
@@ -0,0 +1,13 @@
+import { describe, it, expect} from "@jest/globals";
```

Comment:
test case tests the creation of our starting endpoint well using jest framework. jest framework is useful here as a testing library because we are using react for the frontend of our project. tests the api endpoint that returns Hello as a response, also checks whether the HTTP response status is 200 (OK) and otherwise fails.